From: Adam Conrad <adconrad@ubuntu.com>
Date: Fri, 18 Jul 2014 10:58:46 +0200
Subject: Include sys/xattr.h in sys/capability.h

We include sys/xattr.h before linux/xattr.h to guard against redefinition
of some defines when userspace applications include sys/capability.h and
sys/xattr.h and get linux/xattr.h implicitly included out of order.

Last-Update: 2014-07-26

diff -Naurp libcap2.orig/libcap/include/sys/capability.h libcap2/libcap/include/sys/capability.h
--- libcap2.orig/libcap/include/sys/capability.h
+++ libcap2/libcap/include/sys/capability.h
@@ -27,6 +27,7 @@ extern "C" {
 #define __user
 #endif
 #include <linux/capability.h>
+#include <sys/xattr.h>
 #include <linux/xattr.h>
 
 /*
